tests/rkt_run_pod_manifest_test.go

```diff
@@ -107,6 +107,27 @@ func TestPodManifest(t *testing.T) {
 		cgroup         string
 	}{
 		{
+			// Special characters
+			[]imagePatch{
+				{"rkt-test-run-pod-manifest-read.aci", []string{}},
+			},
+			&schema.PodManifest{
+				Apps: []schema.RuntimeApp{
+					{
+						Name: baseAppName,
+						App: &types.App{
+							Exec:  []string{"/inspect", "--print-msg=\n'\"$"},
+							User:  "0",
+							Group: "0",
+						},
+					},
+				},
+			},
+			true,
+			"'\"$",
+			"",
```

Comment:
I don't know why the test passes on Semaphore but it does not pass on my laptop. I just rebased the patch on my laptop.